docs/index.rst

```diff
@@ -7,6 +7,39 @@
 Welcome to bodhi's documentation!
 =================================
 
+What is Bodhi?
+--------------
+Bodhi is a modular web-based system that facilitates the process of publishing package updates for Fedora. Bodhi is the application that manages updates for Fedora releases. It maintains a single stage of repositories by adding/updating/removing packages. It can be used to create or modify updates and overrides.
+
+Advantages of bodhi
+-------------------
+
+1.  Provides an intuitive interface for developers and release engineers to manage pushing out package updates for multiple version releases.
```

Comment:
I don't think we need to say it's intuitive - keeping the prose short and to the point makes it more readable. Keep in mind that many people reading the home page of a project are going to be quickly skimming it to find out what it is, so we want to be as direct as possible. We don't need to try to convince them to use Bodhi, we just need to inform them what Bodhi is.